Documentation/data-collection.md

```diff
@@ -21,318 +37,495 @@ data:
     - '{__name__="node_role_os_version_machine:cpu_capacity_cores:sum"}'
     #
-    # (@openshift/openshift-team-cluster-manager)
+    # (@openshift/openshift-team-monitoring, @smarterclayton)
+    #
     # node_role_os_version_machine:cpu_capacity_sockets:sum is the total number
     # of CPU sockets in the cluster labeled by master and/or infra node role,
     # os, architecture, and hyperthreading state.
+    #
+    # consumers: (@openshift/openshift-team-cluster-manager)
     - '{__name__="node_role_os_version_machine:cpu_capacity_sockets:sum"}'
     #
+    # (@openshift/openshift-team-olm)
+    #
     # subscription_sync_total is the number of times an OLM operator
     # Subscription has been synced, labelled by name and installed csv
     - '{__name__="subscription_sync_total"}'
     #
-    # olm_resolution_duration_seconds is the duration of a dependency resolution attempt
     # (@openshift/openshift-team-olm)
+    #
+    # olm_resolution_duration_seconds is the duration of a dependency resolution attempt.
     - '{__name__="olm_resolution_duration_seconds"}'
     #
-    # (@openshift/openshift-team-cluster-manager) csv_succeeded is unique to the
-    # namespace, name, version, and phase labels.  The metrics is always
-    # present and can be equal to 0 or 1, where 0 represents that the csv is
-    # not in the succeeded state while 1 represents that the csv is in the
-    # succeeded state.
+    # (@openshift/openshift-team-olm)
+    #
+    # csv_succeeded is unique to the namespace, name, version, and phase
+    # labels.  The metrics is always present and can be equal to 0 or 1, where
+    # 0 represents that the csv is not in the succeeded state while 1
+    # represents that the csv is in the succeeded state.
+    #
+    # consumers: (@openshift/openshift-team-cluster-manager)
     - '{__name__="csv_succeeded"}'
     #
-    # (@openshift/openshift-team-cluster-manager) csv_abnormal represents the reason why
-    # a csv is not in the succeeded state and includes the namespace, name,
-    # version, phase, reason labels. When a csv is updated, the previous time
-    # series associated with the csv will be deleted.
+    # (@openshift/openshift-team-olm)
+    #
+    # csv_abnormal represents the reason why a csv is not in the succeeded
+    # state and includes the namespace, name, version, phase, reason labels.
+    # When a csv is updated, the previous time series associated with the csv
+    # will be deleted.
+    #
+    # consumers: (@openshift/openshift-team-cluster-manager)
     - '{__name__="csv_abnormal"}'
     #
-    # (@openshift/ocs-operator, OCS-storage-management-team) Generic Storage metrics:
-    # cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum gives
-    # the total amount of storage requested by PVCs from a particular storage provisioner in bytes.
+    # (@openshift/ocs-operator, OCS-storage-management-team)
```

Comment:
ocs isn't a valid organization team, maybe [team-ocs-commiters](https://github.com/orgs/openshift/teams/team-ocs-committers). I think we should also remove `OCS-storage-management-team` since it is redundant

Reply:
done